### PR TITLE
fix(documentation): move internet-header, footer and breadcrumbs to raw-components and new header to modules

### PR DESCRIPTION
**Pull request by oliverschuerch**

### CODE REVIEW

File: packages/documentation/src/stories/modules/header/language-switch/language-switch.stories.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { MetaComponent } from '@root/types';
 
 const meta: MetaComponent<HTMLPostLanguageSwitchElement> = {
   id: 'decbb10c-2b39-4f47-b67d-337d8111a3ae',
-  title: 'Components/Header/Language Switch',
+  title: 'Modules/Header/Language Switch',
```
Comment (alizedebray) [inline packages/documentation/src/stories/modules/header/language-switch/language-switch.stories.ts:7]: The language switch is not a module:
```suggestion
  title: 'Components/Language Switch',
```

**Review by gfellerph — APPROVED** [review]

File: packages/documentation/src/stories/modules/header/header.stories.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@ import HeaderMarkup from './components/header.markup';
 
 const meta: MetaComponent = {
   id: 'header',
-  title: 'Raw Components/Header',
+  title: 'Modules/Header/Header',
```
Comment (alizedebray) [inline packages/documentation/src/stories/modules/header/header.stories.ts:7]:
```suggestion
  title: 'Modules/Header',
```

**Review by alizedebray — CHANGES_REQUESTED** [review]
Can you make sure the "Modules" section is initially collapsed?

File: packages/documentation/src/stories/modules/header/language-switch/language-switch.stories.ts (hunk 1 above)
Comment (oliverschuerch) [reply packages/documentation/src/stories/modules/header/language-switch/language-switch.stories.ts:7]: I know, but I thought you want to have it at the same place, where the header docs are. At least this was the case before.

**Review by alizedebray — APPROVED** [review]

**Pull request merged**